### ISSUE TITLE
The latest version v2.3.0 didn't export reduxActions

### ISSUE THREAD

**Issue by functor-dev** [TimboKZ/Chonky#115]
I tried to import `reduxActions` from `chonky/lib/redux/reducers` and failed, seems like something has been broken in **v2.3.0** and **v2.2.0**. But if I try **v2.1.0** then it works.

I checked chonky in node_modules, **it didn't have `lib` folder**, also the file `chonky.cjs.ENVIRONMENT.js` didn't export **reduxActions**.

Is there any way to import reduxActions in v2.3.0?

## Context

I'm using:

- Node.js: _v14.17.6_
- Yarn: _v1.22.11_
- Chonky: _v2.3.0_

## Process

Here are the steps to reproduce:

1. Install latest chonky: `yarn add chonky@latest`
2. Import `import { reduxActions } from 'chonky/lib/redux/reducers';`

## Expected result

Shouldn't throw an error

## Current result

It throws this error:

```
Module not found: Can't resolve 'chonky/lib/redux/reducers' in '/.../src/App.js'
```

## Possible Fix

Downgrade to v2.1.0

## `Error` screenshot

![image](https://user-images.githubusercontent.com/60058060/143766529-f5d54646-dc92-4b1e-9aed-2d42de39799e.png)

**Comment by tuan43** [comment]
getting same issue :(

**Comment by TimboKZ** [comment]
Just released `chonky@2.3.2` which exports `reduxActions`, import like the one shown below should work now.
```js
import { reduxActions } from 'chonky';
```
